frontend/packages/core/src/Feedback/stories/warning.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+// TODO: remove when https://github.com/lyft/clutch/pull/607/files#r512255592 lands
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { action } from "@storybook/addon-actions";
+import type { Meta } from "@storybook/react";
+
+import type { WarningProps } from "../warning";
+import Warning from "../warning";
+
+export default {
+  title: "Core/Warning",
+  component: Warning,
+} as Meta;
+
+const Template = (props: WarningProps) => <Warning {...props} />;
+
+export const Primary = Template.bind({});
+Primary.args = {
+  message: "Informational warning",
+};
+
+export const CloseAction = Template.bind({});
+CloseAction.args = {
+  ...Primary.args,
+  onClose: action("on-close"),
```

Comment:
I wonder if both stories should have an onClose? just b/c both warnings have an **X**

Reply:
Hmmm this might be a good place to chat about the name. The onClose prop is meant to allow for a side-effect during close but it's not a required prop since we may not want to trigger additional logic when the modal disappears. Let me know if you have ideas on how to clarify this :)

Reply:
Thanks for explaining! That makes sense. I think the current name works.